clients/rust/Cargo.toml

```diff
@@ -22,17 +22,17 @@ num-derive = "^0.3"
 num-traits = "^0.2"
 serde = { version = "^1.0", features = ["derive"], optional = true }
 serde_with = { version = "^3.0", optional = true }
-solana-program = "^1.14"
-thiserror = "^1.0"
+solana-program = "~1.18.11"
+thiserror = "1.0.63"
```

Comment:
The Rust client is designed to not require regular Solana program dependency updates.  Can it be left at ^1.14 in the client?